std/experimental/logger/filelogger.d

```diff
@@ -259,7 +259,7 @@ class FileLogger : Logger
     file.close();
 }
 
-@safe unittest
+@trusted unittest
```

Comment:
Are `@trusted` unittests a thing? Why not just `@system`?

Reply:
I suspect the style checker will complain, but if it cannot be safe if should be system